qlib/TableMapper.qm

```diff
@@ -740,12 +771,13 @@ on_error table_mapper.rollback();
 
             hash h;
             foreach hash row in (iter) {
+                # FIXME --PQ 21-Mar-2017
```

Comment:
it is bad comment - I meant "const mappers", which in my terminology means a mapper that contains only constant mappings ("constant" or "runtime", basicly). I'll fix the comment.